storage/tests/test_storage.py

```diff
@@ -0,0 +1,8 @@
+from robocorp import storage
+
+
+class TestLibrary:
+
+    def test_list_assets(self):
+        asset_list = storage.list_assets()
```

Comment:
Simple dummy test that actually works with all the env vars in place. (so the API can be queried)

Reply:
As a note, the vault has a test which runs actually integrated with the cloud (with utilities such as getting `rcc` setup from `devutils`)... I guess it'd be nice to have at least one such integration test here -- see: `vault.tests.robocorp_vault_tests.test_vault_api.test_vault_integrated`

Reply:
Good to know, I prepared some complete working tests now (which take 1min to run -- API optimizations incoming) if the required CR env vars (`RC_...`) are in place.

1. Still, since the current `storage` library infrastructure will be refactored once we introduce the Adapter pattern into this library as well, I find it a little bit unnecessary right now, but at least the tests are XFAILED in CI and XPASSED locally with the right env vars supplied.
2. And I think we can improve the state provisioning during the integrated test runs without blending too much the pure testing part with the env setup.

So I'd place 1 & 2 for an upcoming follow-up PR, as on Tuesday `robocorp-storage` should be released so I can already start its wrapping into `rpaframework`.
Do you think we can afford such a minor tech-debt?